### PR TITLE
Added Cropping, Permute, and multiple activation layers to Tensorflow.Keras

### PR DESCRIPTION
**Pull request by KevinOfCathay**
Layers added:
Cropping1D, Cropping2D, Cropping3D, Permute
ELU, SELU, Softmax, Softsign, Softplus, Exponential, Swish, Tanh, Hard Sigmoid

### CODE REVIEW

**Comment by Oceania2018** [comment]
@KevinOfCathay Thanks for your contribution.
Can you fix the error: https://ci.appveyor.com/project/Haiping-Chen/tensorflow-net/builds/40114211/job/62cap3onelrktb5n ?

**Review by Oceania2018 — APPROVED** [review]
